src/backtrace/mod.rs

```diff
@@ -176,6 +176,7 @@ cfg_if::cfg_if! {
                 unix,
                 not(target_os = "emscripten"),
                 not(all(target_os = "ios", target_arch = "arm")),
+                not(all(target_os = "visionos", target_arch = "arm")),
```

Comment:
Seems unnecessary, there is no 32-bit `arm` architecture for visionos, only `aarch64`.

Reply:
```suggestion
```